nixos/modules/virtualisation/containers.nix

```diff
@@ -564,6 +570,18 @@ in
               '';
             };
 
+            writableAPIVFS = mkOption {
+              type = with types; either bool str;
```

Comment:
Tip: you don't need the assertion above if you use types.enum here instead of str.